web/client/jsapi/MapStore2.js

```diff
@@ -105,6 +105,10 @@ const getInitialActions = (options) => {
 /**
  * MapStore2 JavaScript API. Allows embedding MapStore2 functionalities into
  * a standard HTML page.
+ *
+ * ATTENTION: As of July 2020 a number of MapStore2 plugins (i.e. TOC layer settings, Identify) use react-dock for providing
+ * Dock panel functionality, that assumes that we use the whole window, so the panels won't show up at all or will
+ * not be constrained within the container.
```

Comment:
@offtherailz and @vlt1

please take a look to this https://github.com/geosolutions-it/MapStore2/issues/5143#issuecomment-683708498 about embedded panels configuration

Reply:
@allyoucanmap Yeah, using fluid works. Don't know why though :). Okay then.

Reply:
@vlt1 We introduced the `ms-absolute-dock` className to react-dock to force position absolute instead of fixed to make the left and width properties relative to the root node. It's working with fluid because react-dock is using percentage as unit of measure while without fluid it's trying to calculate the left position using the [window.innerWidth](https://github.com/alexkuz/react-dock/blob/143143e245564f4420baf3403dc5dd59d381625b/src/Dock.js#L366). For this reason before it was under the description panel.